### PR TITLE
Create OLM structure for 15.1.0

### LINKED ISSUE #415: Publish Keycloak Operator 15.1.0 in OperatorHub
### Description

Publish Keycloak Operator 15.1.0 in OperatorHub.io

### Discussion

_No response_

### Motivation

_No response_

### Details

_No response_

### PR DESCRIPTION
**Pull request by jonathanvila**
#415

## Related GH Issue
Closes #415 

## Additional Information
To generate structure to add to the PR to publish in OperatorHub.io

## Verification Steps
1. Build bundle and test catalog
2. deploy it in OCP
3. search for keycloak operator in operators , check it comes from test catalog
4. install it
5. check keycloak version is 15.1.0

## Checklist:
- [ ] Automated Tests
- [ ] [Documentation](https://github.com/keycloak/keycloak-documentation) changes if necessary
- [ ] (If required) Discussed on [Keycloak DEV](https://groups.google.com/forum/#!forum/keycloak-dev)

### CODE REVIEW

File: deploy/olm-catalog/keycloak-operator/15.1.0/manifests/keycloakclients.keycloak.org.crd.yaml
Comment (vmuzikar) [inline deploy/olm-catalog/keycloak-operator/15.1.0/manifests/keycloakclients.keycloak.org.crd.yaml:8]: This needs to be identical with [`keycloak.org_keycloakclients_crd.yaml`](https://github.com/keycloak/keycloak-operator/blob/2deeb25d373c7dbdb67d702244620269502fcacb/deploy/crds/keycloak.org_keycloakclients_crd.yaml).

Comment (jonathanvila) [reply deploy/olm-catalog/keycloak-operator/15.1.0/manifests/keycloakclients.keycloak.org.crd.yaml:8]: yes, which is the problem here ?

Comment (vmuzikar) [reply deploy/olm-catalog/keycloak-operator/15.1.0/manifests/keycloakclients.keycloak.org.crd.yaml:8]: It's not identical. :)
Screenshot 2021-12-14 at 15 03 30

**Review by vmuzikar — APPROVED** [review]
LGTM, thanks @jonathanvila!

Will wait with the merge until after the OperatorHub PRs are merged.

**Comment by jonathanvila** [comment]
@vmuzikar both PRs merged : 
https://github.com/k8s-operatorhub/community-operators/pull/493
https://github.com/redhat-openshift-ecosystem/community-operators-prod/pull/531

Could you please merge this one once you have the time ? Thank you.

**Pull request merged**
